ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -184,16 +184,18 @@ private static void setTransformProperty(View view, ReadableArray transforms) {
 
     if (perspectiveArray.length > PERSPECTIVE_ARRAY_INVERTED_CAMERA_DISTANCE_INDEX) {
       float invertedCameraDistance = (float) perspectiveArray[PERSPECTIVE_ARRAY_INVERTED_CAMERA_DISTANCE_INDEX];
-      if (invertedCameraDistance < 0) {
-        float cameraDistance = -1 / invertedCameraDistance;
-        float scale = DisplayMetricsHolder.getScreenDisplayMetrics().density;
+      if (invertedCameraDistance == 0) {
+        // Default camera distance, before scale multiplier (1280)
+        invertedCameraDistance = 0.00078125f;
```

Comment:
This is the exact value that comes through if you pass a `{perspective: 1280}` value in the transform array on the JS side. When omitting a perspective transform, the value is 0. With this change, the value is forced from 0 to the default 1280 value, forcing the `. setCameraDistance` call.

Not calling `.setCameraDistance` is causing the rendering issue, which is caused by a 0 `invertedCameraDistance`.